.github/workflows/main.yml

```
@@ -0,0 +1,71 @@
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest 
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
```

actionlint

property "python-version" is not defined in object type {}

(expression)